package.json

```diff
@@ -23,6 +24,13 @@
   "devDependencies": {
     "grunt": "^1.0.1",
     "grunt-build-control": "^0.7.1",
-    "grunt-cli": "^1.2.0"
+    "grunt-cli": "^1.2.0",
+    "kalastatic": "^3.1.0"
+  },
+  "dependencies": {
+    "bootstrap-sass": "^3.3.7",
+    "font-awesome": "^4.7.0",
+    "kss-bootstrap": "^4.0.0",
```

Comment:
Yeah I was pretty liberal with how much opinionated build settings were included. We can definitely take some out though, if you think I went too far. I figure if more than half of our sites will use it, then it makes sense. The rest can take it out.